### ISSUE TITLE
Is the goal to build ansible scripts to install all JBoss products?

### ISSUE THREAD

**Issue by alberttwong** [kameshsampath/devops#1]
Is the goal to build ansible scripts to install all JBoss products?  The current Red Hat JBoss stuff in Ansible Galaxy doesn't seem well used.

**Comment by kameshsampath** [comment]
@alberttwong yes, typically eap products along with option to build the docker containers using the ansible playbooks. I never seen the Ansible JBoss playbook

**Comment by alberttwong** [comment]
The Ansible JBoss playbook just installs jboss AS 7.
